frontend/locales/es.ts

```diff
@@ -0,0 +1,868 @@
+import { LocaleMessageObject } from "vue-i18n";
+
+const msgs: LocaleMessageObject = {
+    general: {
+        close: 'Cerrar',
+        submit: 'Enviar',
+        save: 'Guardar',
+        comment: 'Comentar',
+        change: 'Cambiar',
+        donate: 'Donar',
+        continue: 'Continuar',
+        create: 'Crear',
+        delete: 'Eliminar',
+        or: 'O',
+        reset: 'Reiniciar',
+        edit: 'Editar',
+        required: 'Requerido',
+        add: 'Añadir',
+        name: 'Nombre',
+        link: 'Enlace',
+        send: 'Enviar',
+        home: 'Inicio',
+        message: 'Mensaje',
+        refresh: 'Actualizar',
+        confirm: 'Confirmar',
+        error: {
+            invalidUrl: 'Formato de URL inválido',
+        },
+    },
+    hangar: {
+        projectSearch: {
+            query: 'Buscar en {0} proyectos orgullosamente hechos por la comunidad...',
+            relevanceSort: 'Ordenar por importancia',
+            noProjects: 'No hay proyectos. 😢',
+            noProjectsFound: 'Se han encontrado 0 proyectos. 😢',
+        },
+        subtitle: 'Un repositorio de paquetes de Minecraft',
+        sponsoredBy: 'Patrocinado por',
+    },
+    pages: {
+        staffTitle: 'Staff',
+        authorsTitle: 'Autores',
+        headers: {
+            username: 'Nombre',
+            roles: 'Roles',
+            joined: 'Se unió',
+            projects: 'Proyectos',
+        },
+    },
+    nav: {
+        login: 'Inicio de sesión',
+        signup: 'Registro',
+        user: {
+            notifications: 'Notificaciones',
+            flags: 'Marcas',
+            projectApprovals: 'Aprobaciones del proyecyo',
+            versionApprovals: 'Versiones aprobadas',
+            stats: 'Estadísticas',
+            health: 'Salud de Hangar',
+            log: 'Registro de las acciones del usuario',
+            platformVersions: 'Versiones de la plataforma',
+            logout: 'Cerrar sesión',
+            error: {
+                loginFailed: 'Inicio de sesión fallido',
+                invalidUsername: 'Usuario inválido',
+                hangarAuth: "No se puede conectar a HangarAuth",
+                loginDisabled: 'El reguistro esta desactivado temporalmente, intentalo de nuevo más tarde',
+                fakeUserEnabled: 'El usuario falso esta activado. Por tanto, {0} está desactivado',
+            },
+        },
+        createNew: 'Crear nuevo...',
+        new: {
+            project: 'Nuevo proyecto',
+            organization: 'Nueva organización',
+        },
+        hangar: {
+            home: 'Inicio',
+            forums: 'Foros',
+            code: 'Código',
+            docs: 'Documentacion',
+            javadocs: 'JavaDocs',
+            hangar: 'Hangar (Plugins)',
+            downloads: 'Descargas',
+            community: 'Comunidad',
+        },
+    },
+    project: {
+        stargazers: 'Stargazers', //TODO: To be checked/
```

Comment:
`Estrellas`

Reply:
I think I’ll opt for this as there is no specific word for stargazer. Well, actually I don’t even know where does stargazer come from